### ISSUE TITLE
Possible Tiktok DNS Evading

### ISSUE THREAD

**Issue by xRuffKez** [hagezi/dns-blocklists#1445]
### Which domain(s) should be blocked?

direct-http.quic-webcast-proxy-i18n-v3.useast2a.byteglb.com

### Why should these domain(s) be blocked?

Tiktok obviously uses this domain to evade blocked domains.

**Comment by xRuffKez** [comment]
Blocking this Domain des not effect usage of the App.

**Comment by hagezi** [comment]
Possible proxy domains:

```
direct-http.quic-proxy-useast4-v3.useast4.byteglb.com
direct-http.quic-tiktok-proxy-i18n-v3.useast2a.byteglb.com
direct-http.quic-webcast-proxy-gcpsg-v3.gcpsg.byteglb.com
direct-http.quic-webcast-proxy-i18n-v3.useast2a.byteglb.com
direct-http.quic-webcast-proxy-useast4-v3.useast4.byteglb.com
```

**Comment by xRuffKez** [comment]
Close as Wontfix please, blocking this Domain Breaks some functions!

**Comment by xRuffKez** [comment]
I keep testing with the domains blocked above and will post my results.

**Comment by hagezi** [comment]
As it looks to me, it is not a bypass. Some API functions seem to be blocked by this.

**Comment by xRuffKez** [comment]
I think these domains act like some kind of CDN maybe

**Comment by xRuffKez** [comment]
My family members still don't have issues with the domains blocked

**Comment by hagezi** [comment]
Not here either so far. I will block them in the Ultimate and in the aggressive version of the native TikTok tracker list. Let's see if there are any reports on this.

**Comment by hagezi** [comment]
@xRuffKez included in ultimate.

**Comment by xRuffKez** [comment]
Thank you, i will keep an eye on the functionality.

**Comment by hagezi** [comment]
@xRuffKez If the proxy domains are blocked, the app often crashes under iOS. I also had these reports from the family. Furthermore, it is sometimes displayed that the video could not be loaded. Unblocking the proxy domains solves the problem.

I will remove the domains from the lists again.
